src/domain/event/mutation.ts

```diff
@@ -106,6 +114,14 @@ export const MUTATION_EVENT = gql`
             photographerName
             altText
           }
+          offers {
+            price {
+              en
+              fi
+              sv
+            }
+            isFree
+          }
```

Comment:
offerFields fragment could be used if it doesn't add overhead in the backend?

otherwise I think a new fragment could be created here because this is repeated twice :)

Reply:
That's true